data/en/ukis_0001.json

```diff
@@ -4565,129 +4206,151 @@
                             }]
-                        }]
-                    }
-                ]
-            }]
+                        },
+                        {
+                            "id": "block2527",
+                            "type": "Question",
+                            "questions": [{
+                                "id": "question2527",
+                                "title": "Would {{ metadata['ru_name'] }} be willing to be contacted by telephone by the Department for Business, Energy and Industrial Strategy or its appointed agents, to ask some further questions about innovation?",
+                                "type": "General",
+                                "answers": [{
+                                    "id": "answer3275",
+                                    "mandatory": false,
+                                    "type": "Radio",
+                                    "label": "",
+                                    "description": "",
+                                    "options": [{
+                                            "label": "Yes",
+                                            "value": "Yes"
+                                        },
+                                        {
+                                            "label": "No",
+                                            "value": "No"
+                                        }
+                                    ]
+                                }]
+                            }]
+                        }
+                    ]
+                },
+                {
+                    "id": "summary-group",
+                    "title": "Summary",
+                    "blocks": [{
+                        "type": "Summary",
+                        "id": "summary-block",
+                        "collapsible": true
+                    }]
+                }
+            ]
+        }
+    ],
+    "theme": "ukis",
+    "navigation": {
+        "visible": true
+    },
+    "metadata": [{
+            "name": "user_id",
+            "validator": "string"
         },
         {
-            "id": "summary-section",
-            "type": "Summary",
-            "title": "Check your answers and submit",
-            "groups": [{
-                "id": "summary-group",
-                "title": "Summary",
-                "blocks": [{
-                    "type": "Summary",
-                    "id": "summary-block",
-                    "collapsible": true
-                }]
-            }]
+            "name": "period_id",
+            "validator": "string"
+        },
+        {
+            "name": "ru_name",
+            "validator": "string"
+        },
+        {
+            "name": "trad_as",
+            "validator": "optional_string"
+        },
+        {
+            "name": "ref_p_start_date",
+            "validator": "date"
+        },
+        {
+            "name": "ref_p_end_date",
+            "validator": "date"
```

Comment:
Double checked with the business via Jordan, we don't need these two bits of metadata in here.